openweathermap-ios/SceneDelegate.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  SceneDelegate.swift
+//  openweathermap-ios
+//
+//  Created by Felipe Weber on 11/03/20.
+//  Copyright © 2020 Lucas Montano. All rights reserved.
+//
+
+import UIKit
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
```

Comment:
```suggestion
final class SceneDelegate: UIResponder, UIWindowSceneDelegate {
```

To improve compiler speed.